cocos/core/scene-graph/scene-globals.ts

```diff
@@ -28,21 +28,30 @@ import { ccclass, visible, type, displayOrder, readOnly, slide, range, rangeStep
 import { EDITOR } from 'internal:constants';
 import { TextureCube } from '../assets/texture-cube';
 import { CCFloat, CCBoolean, CCInteger } from '../data/utils/attribute';
-import { Color, Quat, Vec3, Vec2, color, Vec4 } from '../math';
+import { Color, Quat, Vec3, Vec2, Vec4 } from '../math';
 import { Ambient } from '../renderer/scene/ambient';
 import { Shadows, ShadowType, PCFType, ShadowSize } from '../renderer/scene/shadows';
 import { Skybox } from '../renderer/scene/skybox';
 import { Fog, FogType } from '../renderer/scene/fog';
 import { Node } from './node';
 import { legacyCC } from '../global-exports';
-import { Asset } from '..';
 import { Root } from '../root';
-import { normalize } from '../asset-manager/helper';
 
 const _up = new Vec3(0, 1, 0);
 const _v3 = new Vec3();
 const _qt = new Quat();
 
+// Normalize HDR color
+const normalizeHDRColor = (color : Vec4) => {
+    const intensity = 1.0 / Math.max(Math.max(color[0], color[1]), color[2]);
+    if (intensity < 1.0) {
+        for (let i = 0; i < 3; ++i) {
+            color[i] *= intensity;
+        }
+    }
+}
+const clampColor = (x: number) => Math.min(x * 255, 255);
```

Comment:
Make `normalizeHDRColor` and `clampColor` as script level internal tool functions, if no outside classes need this kind of tool functions, then we can use this kind of declaration.
If other classes also need them, we need to put them into tool or utils module